css/style.css

```
@@ -2269,3 +2269,398 @@ body.console-active .main {
+/* ========== Hex Editor Container ========== */
+.hexeditor-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 1001;
+  background-color: #1c1c1c;
+  display: flex;
+  flex-direction: column;
+  font-family: "SFMono-Regular", Consolas, "Liberation Mono", Menlo, Courier, monospace;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)